src/Templates/AdminPanel/Bit.AdminPanel/Web/App.razor

```diff
@@ -36,17 +36,11 @@
     private async Task OnNavigateAsync(NavigationContext args)
     {
 #if BlazorWebAssembly
-        /* To lazy load an assembly based on the current route, follow this:
-        1- Add the BlazorWebAssemblyLazyLoad tag to the Web project file like:
-        <BlazorWebAssemblyLazyLoad Condition=" '$(BlazorMode)' == 'BlazorWebAssembly' " Include="Sample.dll" />
-        2- Add the assembly to the lazyLoadedAssemblies as follows:
-        if (args.Path.Contains("sample"))
+        if (args.Path.Contains("analytics"))
```

Comment:
we won't have any charts on the home page?

Reply:
Otherwise, lazy loading of these three DLLs does not make sense

Reply:
I think we will with the new design, so we need to keep that in mind too

Reply:
Yes that's right